composer.json

```diff
@@ -10,25 +10,34 @@
         }
     ],
     "require": {
-        "php": ">=5.6.0",
-        "symfony/framework-bundle": "~2.7|~3.0",
-        "twig/twig": ">=1.12,<2.0-dev",
-        "doctrine/annotations": "~1.0"
+        "php": "^7.2",
+        "symfony/framework-bundle": "^3.4.31|~4.3.4",
```

Comment:
Please add Symfony 4.2 as well

Reply:
@elbcoast Symfony v4.2 not maintained anymore 
https://symfony.com/releases